app/src/main/java/com/example/easysurvey/dao/IEasySurveyDAO.kt

```diff
@@ -0,0 +1,10 @@
+package com.example.easysurvey.dao
+
+//dont forget to add an import for it
+interface IEasySurveyDAO {
+    //dont forget json from url
+    @GET("")
+    //change list if this isn't name of it
```

Comment:
Big no-no: do not put conversation in source code comments.  That's clutter.  Put the conversation here, in the Files Changed section of the pull request, where you can have a dialog with other developers.